### PR TITLE
small adjustments in the documentation

### PR DESCRIPTION
**Pull request by hartmut**
minor changes in the specs book

### CODE REVIEW

File: book/src/05_storages.md [hunk 1]
```diff
@@ -52,16 +52,16 @@ fairly well.
 
 ## `DenseVecStorage`
 
-This storage uses two `Vec`s, one containing the actual data and the other
+This storage uses two `Vec`\`s, one containing the actual data and the other
```
Comment (torkleyy) [inline book/src/05_storages.md]: Not a native speaker, but this change seems odd to me.

File: book/src/07_parallel_join.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ method call can be exchanged for `par_join`:
     }
 ```
 
-The `par_join` method produces a type implementing `rayon`s `ParallelIterator`
+The `par_join` method produces a type implementing [`rayon`\`s `ParallelIterator`][ra]
```
Comment (torkleyy) [inline book/src/07_parallel_join.md]: Wouldn't you use `'` instead of ` here?

File: book/src/05_storages.md (hunk 1 above)
Comment (jonas-schievink) [reply book/src/05_storages.md]: Yeah this was fine before the change

File: book/src/07_parallel_join.md (hunk 2 above)
Comment (hartmut) [reply book/src/07_parallel_join.md]: ah ok, I am not a natural speaker too, so sorry for the misspelling

**Comment by torkleyy** [comment]
Thank you!

bors r+

**Comment by torkleyy** [comment]
Seriously?

bors r+

**Pull request merged**